transpire/cli/image.py

```diff
@@ -10,7 +12,45 @@ def commands(**kwargs):
 @commands.command()
 def build(**kwargs):
     """build defined images"""
-    raise NotImplementedError("Not yet implemented!")
+
+    # ensure we have buildkit installed
+    if shutil_which("buildctl") is None:
+        raise RuntimeError("You must install Buildkit to use this script.")
+
+    # get remote and additional tags (like one by commit)
+    remote: str  = kwargs.get('registry', None)
```

Comment:
We should throw an error instead of setting a default value (we need some remote to push to, if we are pushing).